.github/.cache-key

```diff
@@ -0,0 +1 @@
+1
```

Comment:
This is basically a "last CI incident" counter 😂

Reply:
Absolutely 🤣